### PR TITLE
Added more locals to tvpassport

### PR DESCRIPTION
**Pull request by ajanes3**
Updated WFTYDT1
Updated WFTYDT2
Added WFUTDT1
Added WJACDT1 through 3
Added WMBCDT1, 6, and 7
Added WPVIDT1
Added WCSHDT1 through 3
Added WGMEDT1
Added WHECDT1
Added WICUDT1
Added WMTWDT1 and 2
Added WMURDT1 and 2
Added WNNEDT1
Added WPFODT1
Added WPXTDT1
Added WPXUDT1

### CODE REVIEW

**Review by PopeyeTheSai10r — DISMISSED** [review]

**Comment by ajanes3** [comment]
Bah, I dismissed your review. I won't make any more changes for now.

File: sites/tvpassport.com/tvpassport.com.channels.xml [hunk 1]
```diff
@@ -405,19 +408,25 @@
     <channel lang="en" xmltv_id="WFSBDT3.us" site_id="laff-wfsb3-hartford-ct/16153">Laff (WFSB-DT3) Hartford, CT</channel>
     <channel lang="en" xmltv_id="WFSBDT4.us" site_id="cbs-wfsb4-fairfield-county-hartfod-ct/14723">CBS (WFSB-DT4) Fairfield County, CT</channel>
     <channel lang="en" xmltv_id="WFTVDT1.us" site_id="abc-wftv-orlando-fl/2494">ABC (WFTV) Orlando, FL</channel>
-    <channel lang="en" xmltv_id="WFTYDT1.us" site_id="unimas-wfty-smithtown-ny/1961">UniMás (WFTY) Smithtown, NY</channel>
+    <channel lang="en" xmltv_id="WFTYDT1.us" site_id="unimas-wfty-smithtown-ny/1961">True Crime Network (WFTY) Smithtown, NY</channel>
```
Comment (PopeyeTheSai10r) [inline sites/tvpassport.com/tvpassport.com.channels.xml:411]: This should be UniMás.

File: sites/tvpassport.com/tvpassport.com.channels.xml [hunk 2]
```diff
@@ -599,7 +620,8 @@
     <channel lang="es" xmltv_id="KVEADT1.us" site_id="telemundo-kvea-los-angeles-ca/2611">Telemundo (KVEA) Los Angeles, CA</channel>
     <channel lang="es" xmltv_id="W20CQD2.us" site_id="esperanza-w20cqd2-hempstead-ny/16437">Esperanza (W20CQ-D2) Hempstead, NY</channel>
     <channel lang="es" xmltv_id="WASALD1.us" site_id="estrella-wasald-port-jervis-ny/11071">Estrella (WASA-LD) Port Jervis, NY</channel>
-    <channel lang="es" xmltv_id="WFTYDT2.us" site_id="uni-wftydt2-new-york-ny/1962">UNI (WFTY-DT2) New York, NY</channel>
+    <channel lang="es" xmltv_id="WFTYDT2.us" site_id="uni-wftydt2-new-york-ny/1962">UniMás (WFTY-DT2) New York, NY</channel>
```
Comment (PopeyeTheSai10r) [inline sites/tvpassport.com/tvpassport.com.channels.xml:623]: Database have this as Univision - https://iptv-org.github.io/?q=WFTYDT2.us

**Review by freearhey — APPROVED** [review]

File: sites/tvpassport.com/tvpassport.com.channels.xml (hunk 1 above)
Comment (ajanes3) [reply sites/tvpassport.com/tvpassport.com.channels.xml:411]: https://en.wikipedia.org/wiki/WFTY-DT

Wikipedia has DT1 (67.1) as True Crime Network.

File: sites/tvpassport.com/tvpassport.com.channels.xml (hunk 2 above)
Comment (ajanes3) [reply sites/tvpassport.com/tvpassport.com.channels.xml:623]: https://en.wikipedia.org/wiki/WFTY-DT

Wikipedia has DT2 (67.2) as UniMás. (see other comment for relevant quote)

Comment (ajanes3) [reply sites/tvpassport.com/tvpassport.com.channels.xml:623]: https://github.com/iptv-org/database/pull/2164

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**